csrc/executor_kernel_arg.h

```diff
@@ -207,35 +209,36 @@ struct TensorArgAbstract : ArgAbstract {
   std::string toString() const override;
 };
 
-template <typename TENSOR_TYPE, typename nvfuser_index_t>
-// NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
+template <typename TENSOR_TYPE>
 struct TensorArg : public TensorArgAbstract {
   TENSOR_TYPE instance_;
-  // TODO: this is ugly, we should be extracting data type from `instance_`
```

Comment:
This is done at https://github.com/NVIDIA/Fuser/pull/159/files#diff-108f0d9f28a8c37e5ea372fb5eef2659152a439ec8bf90f023e791c798c4838eR251